app/index.ejs

```diff
@@ -1,201 +1,372 @@
+      <div class="container">
+        <div class="row">
+          <div class="col-sm-8">
+            <ul class="list-inline">
+              <li>
+                <a href="https://www.cru.org/us/en/about/privacy.html" translate
+                  >Privacy Policy</a
+                >
+              </li>
+              <li><a href="help/" translate>Help</a></li>
+              <% if(!htmlWebpackPlugin.options.prod) { %>
+              <li>
+                <a ng-href="http://localhost:9000/auth/{{ crsToken }}"
+                  >Local Dev Environment</a
+                >
+              </li>
+              <% } %>
+            </ul>
+          </div>
+          <div class="col-sm-4 text-right">
+            <span
+              >&copy; {{ year }} |
+              <a href="http://www.cru.org" target="_blank">Cru</a></span
+            >
+          </div>
+        </div>
       </div>
-    </div>
-  </div>
-</footer>
-
-<footer id="frontfooter" ng-if="globalPage.type === 'registration'">
-  <div class="container">
-    <span>&copy; {{year}} | <translate>Created with Event Registration Tool, powered by</translate> <a href="http://www.cru.org" target="_blank">Cru</a></span>
-  </div>
-</footer>
+    </footer>
 
-<% if(htmlWebpackPlugin.options.prod) { %>
-<script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+    <footer id="frontfooter" ng-if="globalPage.type === 'registration'">
+      <div class="container">
+        <span
+          >&copy; {{ year }} |
+          <translate
+            >Created with Event Registration Tool, powered by</translate
+          >
+          <a href="http://www.cru.org" target="_blank">Cru</a></span
+        >
+      </div>
+    </footer>
 
-  ga('create', 'UA-325725-55', 'auto');
-</script>
-<% } %>
+    <% if(htmlWebpackPlugin.options.prod) { %>
+    <script>
+      (function(i, s, o, g, r, a, m) {
+        i['GoogleAnalyticsObject'] = r;
+        (i[r] =
+          i[r] ||
+          function() {
+            (i[r].q = i[r].q || []).push(arguments);
+          }),
+          (i[r].l = 1 * new Date());
+        (a = s.createElement(o)), (m = s.getElementsByTagName(o)[0]);
+        a.async = 1;
+        a.src = g;
+        m.parentNode.insertBefore(a, m);
+      })(
+        window,
+        document,
+        'script',
+        '//www.google-analytics.com/analytics.js',
+        'ga',
+      );
 
-<script type="text/javascript">window._satellite && _satellite.pageBottom();</script>
+      ga('create', 'UA-325725-55', 'auto');
+    </script>
 
-<%=htmlWebpackPlugin.files.webpackManifest%>
-</body>
+    <script type="text/javascript">
+      window._satellite && _satellite.pageBottom();
```

Comment:
@brenton05, this doesn't need to be called anymore with Launch right? I put in the if statement that only runs in prod. We'll remove it when we add the prod scripts.

Reply:
referring to line 197? If so, it does not if we are using the launch script I gave you but again, only on stage. Prod will still need the pageBottom call.

Reply:
K this code should be good then. It'll only call pageBottom on prod.